ansible/roles/schulcloud-server-core/templates/secret.yml.j2

```diff
@@ -63,4 +63,10 @@ data:
   SECRET_ES_MERLIN_PW: "{{ SECRET_ES_MERLIN_PW|default("", true) | b64encode }}"
   SECRET_ES_MERLIN_COUNTIES_CREDENTIALS: "{{ SECRET_ES_MERLIN_COUNTIES_CREDENTIALS|default("", true) | b64encode }}"
 {% endif %}
+{% endif %}
+
+  # Nexboard
+{% if FEATURE_NEXBOARD_ENABLED is defined and FEATURE_NEXBOARD_ENABLED == "true" %}
```

Comment:
maybe `=== "true" ` would be more save?

Reply:
guess that is not the same in ansible, we use same everywhere

Reply:
ok